src/data/complex/basic.lean

```diff
@@ -283,6 +283,12 @@ by ext; simp [neg_div]
 @[simp] lemma conj_div (z w : ℂ) : conj (z / w) = conj z / conj w :=
 by rw [division_def, conj_mul, conj_inv]; refl
 
+@[simp] lemma div_I (z : ℂ) : z / I = -(z * I) :=
```

Comment:
Should the RHS be `- (z * I)` or `- z * I`?

Reply:
-(z*I) is simp normal.

Reply:
Ok, thanks